asciidoc/courses/aura-dashboards/modules/2-understand-your-data/lessons/2-build-with-ai/lesson.adoc

```diff
@@ -0,0 +1,167 @@
+= Building dashboards with AI
+:type: lesson
+:order: 2
+
+
+
+In the previous lessons, you learned how to execute Cypher queries using the Query tab and explored your data using the Explore tab.
+
+In this lesson, you will learn how to:
+
+* Create interactive dashboards using Aura's integrated dashboard tool
+* Build dashboard cards using natural language AI prompts
+* Add filters to make dashboards dynamic
+
+
+ 
+== Understanding dashboard tools
+
+A dashboard tool transforms raw data into visual representations that anyone can understand, regardless of their technical background.
+
+Think of dashboards as the "storefront window" for your graph data - they display the most important insights in an accessible, visual format that business users can interact with without writing code.
+
+In Aura, you have access to an integrated AI tool that allows you to quickly prototype and build production-ready dashboards.
```

Comment:
```suggestion
In Aura Dashboards, you have access to an integrated AI tool that allows you to quickly prototype and build production-ready dashboards.
```
There are other AI tools withing the Aura console, so to avoid confusing it's better to be specific.